### DISCUSSION TITLE
[feature] ed25519 support

### DISCUSSION THREAD

**Discussion by t3dium** [FiloSottile/mkcert#477] [Ideas]
faster and considered as secure, if not more, than rsa and ecdsa

**Comment by paepckehh** [comment]
Hi, 
yes, agree - the curve25519 is great and a big sucess everywhere (see X25519 default kex for TLS13).  

But still, Ed25519 based certificates are still not supported in Chrome/Firefox, even if the backend crypto-libs (boringSSL/nss) support it. Its politics, not bits/code. 

* [golang/chrome](https://github.com/golang/go/issues/47681)
* [letsencrypt](https://community.letsencrypt.org/t/ed25519-acme-account-support/93459)
* [stackexchange](https://security.stackexchange.com/questions/236931/whats-the-deal-with-x25519-support-in-chrome-firefox)

I would love to use mkcert polularity and push an internal diff as PR for mkcert - with support for Ed25519. Put some pressue on it! But I'm afraid that a tool that targets none-experts, like mkcert, would produce some bug reports.

Not sure about mkcert team position. 

Here is a ed25519 enabled version
(preview/showcase/temporary-only/fork)

[github.com/paepckehh/mkcert-delux](https://github.com/paepckehh/mkcert-delux)
[github.com/paepckehh/mkcert-delux/releases](https://github.com/paepckehh/mkcert-delux/releases)
```
go install paepcke.de/mkcert-delux@latest
```  
Michael 
(edit: add showcase)

**Comment by crisdosaygo** [comment]
This probably implies RSA is broken already right? Yet ED25519 is not, hence the pressure to continue using an interceptable encryption.
